### ISSUE TITLE
Can we please make this text bold or red or both or just bigger more in your face please? 

Note :This does not replace the existing TSMS sign repair/maintenance documentation requirements.

Please?

### ISSUE THREAD

**Issue by HATSBot** [WSDOT/HATS#1243]
* **Device Name:** HQD9760003
* **Device Model:** iPad 5
* **Device System Version:** 12.1.1
* **Application Version:** HATS-iOS 4.0.4
* **Name:** Peter Burkhard
* **Has Attachments:** True

* **Feedback:** Can we please make this text bold or red or both or just bigger more in your face please? 

Note :This does not replace the existing TSMS sign repair/maintenance documentation requirements.

Please?

* **Details With Attachments:**  <a href='http://hatsprod.wsdot.loc/Maintenance/management/activities/Feedback/GetMobileDeviceFeedback?mobileFeedbackId=3298'>View Details And Attachments</a>

**Comment by PeterBurkhard** [comment]
![image-1 copy](https://user-images.githubusercontent.com/15268732/51269073-62723d00-1976-11e9-944c-52c44f629659.jpeg)
![img_0640](https://user-images.githubusercontent.com/15268732/51269101-7453e000-1976-11e9-97a2-a941a9b32599.PNG)

**Comment by jhudsonWA** [comment]
I would only do it if all help text got the same treatment, or we defined a new type of ImportantHelpText or something like that that made it stand out more. I don't really want to make single one off changes to the forms.

**Comment by joeschmit** [comment]
@jhudsonWA Agree as well, I don't like one offs. I would however second your option by having one or two help text types such as "Warning Help Text (red bold italic)"

**Comment by jhudsonWA** [comment]
Alright, we will work on that for next release. Maybe even 4.1 if I can get it in today or tomorrow.

**Comment by jhudsonWA** [comment]
Will be in next beta (web and iOS). For now I just added support for Warning Help Text, which is always red bold and italic. Support is on web:

<img width="641" alt="image" src="https://user-images.githubusercontent.com/702446/51289335-9704e980-19b4-11e9-8978-da6fa429ce1c.png">

 and ios:

<img width="796" alt="image" src="https://user-images.githubusercontent.com/702446/51289965-26ab9780-19b7-11e9-8843-32b0f4f9c16a.png">


The form builder has also been updated to allow for Warning Help Texts to be added.
